pyspark/bigdl/nn/layer.py

```diff
@@ -3232,21 +3259,6 @@ def __init__(self,
                                           index)
 
 
-class Sigmoid(Layer):
```

Comment:
why delete this?

Reply:
Not delete. Just remove it to the place before LSTM. 
Since in LSTM ut after adding activation paramater, I need Sigmoid() to create it. But if Sigmoid is defined after LSTM, then it will cause a notfound error.